app/models/job.rb

```diff
@@ -1,4 +1,18 @@
 class Job < ActiveRecord::Base
   validates :title, :level_of_interest, presence: true
   belongs_to :company
+  belongs_to :category
+  has_many :comments
+
+  def self.level_of_interest
```

Comment:
i don't see tests for these custom model methods. don't forget to test every method you write!